prov/efa/src/rdm/efa_rdm_ep_fiops.c

```diff
@@ -1646,7 +1646,7 @@ int efa_rdm_ep_check_qp_in_order_aligned_128_bytes(struct efa_rdm_ep *ep,
 	/* Create a dummy qp for query only */
 	efa_rdm_ep_construct_ibv_qp_init_attr_ex(ep, &attr_ex, ibv_cq_ex, ibv_cq_ex);
 
-	ret = efa_qp_create(&qp, &attr_ex);
+	ret = efa_qp_create(&qp, &attr_ex, ep->base_ep.info->tx_attr->tclass);
```

Comment:
this is only a trial QP that query the in order aligned 128 bytes capability, no need to specify tclass here. Can use FI_TC_UNSPEC.